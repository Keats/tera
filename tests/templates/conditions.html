<html>
  <head>
    <title>{{ product.name }}</title>
  </head>
  <body>
    <h1>{{ product.name }} - {{ product.manufacturer }}</h1>
    <p>{{ product.summary }}</p>
    <p>Â£{{ product.price * 1.20 }} (VAT inc.)</p>
    {% if friend_reviewed %}
      <p>Look at reviews from your friends {{ username }}</p>
      {% if number_reviews > 10 or show_more %}
        <p>See all reviews</p>
      {% elif number_reviews == 1 %}
        <p>Only one review</p>
      {% endif %}
    {% else %}
      {% if this_doesnt_exist %}
        Oh no!
      {% else %}
        <p>None of your friend reviewed this product</p>
      {% endif %}
    {% endif %}
    <button>Buy!</button>
  </body>
</html>
