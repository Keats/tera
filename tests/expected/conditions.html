<html>
  <head>
    <title>Moto G</title>
  </head>
  <body>
    <h1>Moto G - Motorala</h1>
    <p>A phone</p>
    <p>Â£120 (VAT inc.)</p>
      <p>Look at reviews from your friends bob</p>
        <p>See all reviews</p>
    <button>Buy!</button>
  </body>
</html>
